.gitattributes

```diff
@@ -3,5 +3,5 @@ travis.zip.enc merge=ours
 README.md merge=ours
 src/app/customHelpers.js merge=ours
 src/app/components/media/MediaStatus.js merge=ours
-test/spec/status_spec.rb merge=ours
```

Comment:
let's remove it and handle the Travis stuff separately (e.g., using one VM for Check and one VM for Bridge)